src/SFA.DAS.Commitments.Support.SubSite/Views/Apprenticeships/Index.cshtml

```diff
@@ -0,0 +1,193 @@
+
+
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Training</h2></th>
+                    </tr>
+                </thead>
+            </table>
+
+            <table class="responsive twoColumn">
+                <tbody>
+                    <tr id="employer">
+                        <td>Legal entity</td>
+                        <td>@Model.LegalEntity</td>
+                    </tr>
+                    <tr id="trainingProvider">
+                        <td>Training provider</td>
+                        <td>@Model.TrainingProvider</td>
+                    </tr>
+                    <tr id="trainingProvider">
+                        <td>UKPRN</td>
+                        <td>@Model.UKPRN</td>
+                    </tr>
+
+                    <tr>
+                        <td>Apprenticeship training course</td>
+                        <td>@Model.Trainingcourse</td>
+                    </tr>
+                    <tr>
+                        <td>Apprenticeship code<br>(needed for ILR)</td>
+                        <td>@Model.ApprenticeshipCode</td>
+                    </tr>
+
+
+                </tbody>
+            </table>
+
+
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Dates</h2></th>
+                    </tr>
+                </thead>
+            </table>
+
+            <table class="responsive twoColumn">
+                <tbody>
+
+                    <tr>
+                        <td>AS training start date</td>
+                        <td>
+                            @{
+                                var startDate = Model.DasTrainingStartDate.HasValue ? Model.DasTrainingStartDate.Value.ToGdsFormatWithoutDay() : "";
+                            }
+
+                            @startDate
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>AS training end date</td>
+                        <td>
+                            @{
+                                var endDate = Model.DasTrainingEndDate.HasValue ? Model.DasTrainingEndDate.Value.ToGdsFormatWithoutDay() : "";
+                            }
+
+                            @endDate
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+
+
+            <table class="responsive twoColumn">
+                <thead>
+                    <tr>
+                        <th colspan="2"> <h2 class="heading-medium">Payment</h2></th>
+                    </tr>
+                </thead>
+            </table>
+
+            <table class="responsive twoColumn">
+                <tbody>
+                    <tr>
+                        <td>Cost</td>
+                        <td>@Model.TrainingCost.ToGdsCurrencyFormat()</td>
```

Comment:
TrainingCost is nullable, so we should handle that here.
As this is a support console, we shouldn't assume all values are present, as the user will be viewing entities associated with an issue, so we should expect and handle bad data.

Reply:
Already handled by the extension method